src/controllers/touristCenterController.js

```diff
@@ -0,0 +1,120 @@
+import db from "../services/AdminServices/touristCenterService";
+import { validation, validateId } from "../validation/touristCenterValidation";
+
+/**
+ * @class touristCenterController
+ * @description create, read, update & delete touristcenter
+ * @exports touristCenterController
+ */
+export default class touristCenterController {
+  /**
+   * @param {object} req - The user request object
+   * @param {object} res - The user response object
+   * @returns {object} Success message
+   */
+  static async addTouristCenter(req, res) {
+    try {
```

Comment:
now, when adding countryId, don't include it with the body include it as a header, so the route will be `admin/tourist-center/:countryId`